course_content/notebooks/iris_intro.ipynb

```diff
@@ -1002,9 +1002,9 @@
    },
    "outputs": [],
    "source": [
-    "print raw_cubes[0]\n",
-    "print\n",
-    "print raw_cubes[1]"
+    "print(raw_cubes[0])\n",
+    "print()\n",
```

Comment:
I just don't like an empty print call... [further down](https://github.com/SciTools/courses/pull/129/files#diff-6a5bf1866442f8f0072cf85e31079d02R1108) we use a line of dashes to perform the same job; we could replicate that here.